client/my-sites/plugins/controller.js

```diff
@@ -149,6 +152,40 @@ function renderPluginsBrowser( context, siteUrl ) {
 	);
 }
 
+function renderNoManageWarning() {
+	let site = sites.getSelectedSite();
+	ReactDom.render(
+		React.createElement( JetpackManageErrorPage, {
+			site: site,
+			template: 'optInManage',
+			title: i18n.translate( 'Oh no! We can\'t automatically install your new plugins.' ),
+			section: 'plugins',
+			illustration: '/calypso/images/jetpack/jetpack-manage.svg',
+		} ),
```

Comment:
Does it make sense to include a feature-example (https://wpcalypso.wordpress.com/devdocs/design/feature-example) here as we do with some other JetpackManageErrorPages?